qutip/solver/integrator/krylov.py

```diff
@@ -28,7 +28,11 @@ class IntegratorKrylov(Integrator):
     supports_blackbox = False
     method = 'krylov'
 
-    def _prepare(self):
+    def __init__(self, solver):
+        self.system = solver.H
+        self._options = self.integrator_options.copy()
+        self.options = solver.options
```

Comment:
What is the difference between `solver.H` and `solver.RHS` here? What does this `__init__` set `self._options` and not others?